framework/data/DataFilter.php

```diff
@@ -388,12 +388,12 @@ public function setErrorMessages($errorMessages)
     protected function defaultErrorMessages()
     {
         return [
-            'invalidFilter' => Yii::t('yii-rest', 'The format of {filter} is invalid.'),
-            'operatorRequireMultipleOperands' => Yii::t('yii-rest', "Operator '{operator}' requires multiple operands."),
-            'unknownAttribute' => Yii::t('yii-rest', "Unknown filter attribute '{attribute}'"),
-            'invalidAttributeValueFormat' => Yii::t('yii-rest', "Condition for '{attribute}' should be either a value or valid operator specification."),
-            'operatorRequireAttribute' => Yii::t('yii-rest', "Operator '{operator}' must be used with a search attribute."),
-            'unsupportedOperatorType' => Yii::t('yii-rest', "'{attribute}' does not support operator '{operator}'."),
+            'invalidFilter' => Yii::t('yii-data', 'The format of {filter} is invalid.'),
+            'operatorRequireMultipleOperands' => Yii::t('yii-data', "Operator '{operator}' requires multiple operands."),
+            'unknownAttribute' => Yii::t('yii-data', "Unknown filter attribute '{attribute}'"),
+            'invalidAttributeValueFormat' => Yii::t('yii-data', "Condition for '{attribute}' should be either a value or valid operator specification."),
+            'operatorRequireAttribute' => Yii::t('yii-data', "Operator '{operator}' must be used with a search attribute."),
+            'unsupportedOperatorType' => Yii::t('yii-data', "'{attribute}' does not support operator '{operator}'."),
```

Comment:
there are translation files to be renamed:

```
$ find messages/ |grep rest
messages/de/yii-rest.php
messages/hy/yii-rest.php
```

Reply:
Also I don't see why we have a separate category for these. why not use `yii` here too? what is different?

Reply:
I don't think anything is...